api/models/usersChannels.js

```diff
@@ -63,15 +63,13 @@ const createMemberInChannel = (channelId: string, userId: string, token: boolean
 
   return db
     .table('usersChannels')
-    .getAll(userId, { index: 'userId' })
-    .filter({ channelId })
+    .getAll([userId, channelId], { index: 'userIdAndChannelId' })
     .run()
     .then(result => {
       if (result && result.length > 0) {
         return db
           .table('usersChannels')
-          .getAll(userId, { index: 'userId' })
-          .filter({ channelId, isBlocked: false })
+          .getAll([userId, channelId], { index: 'userIdAndChannelId' })
```

Comment:
I think we still want the `isBlocked` filter here right? Otherwise someone could spoof joining a channel via a direct POST request